WrightTools/kit/_array.py

```diff
@@ -270,11 +270,10 @@ def valid_index(index, shape):
     tuple
         Valid index.
     """
+    # TODO: fails in case of shape (25, 1), index slice(None)
```

Comment:
Is this still true? Either remove TODO or open an issue

Reply:
yes, this is actually a big problem---embarising that I forgot about it

don't merge for now